### PR TITLE
Changed for new schema - untested

### PR DESCRIPTION
**Pull request by Vaspra**
Intend to build upon this slightly with tag features

### CODE REVIEW

File: zegami_sdk/annotation.py [hunk 1]
```diff
@@ -7,16 +7,18 @@
 import base64
 import io
 import os
-
 import numpy as np
 from PIL import Image
 
+from zegami_ai.mrcnn.utils import get_bool_mask_bounds
```
Comment (Vaspra) [inline zegami_sdk/annotation.py:13]: I may remove this and copy the function over to break the dependency on zegami-ai

Comment (Vaspra) [reply zegami_sdk/annotation.py]: Done in latest commit

File: zegami_sdk/annotation.py [hunk 2]
```diff
@@ -98,7 +99,9 @@ class AnnotationMask(_Annotation):
     obtained automatically, but this is slow and can cause unnecessary
     re-downloading of data. """
 
-    TYPE = 'mask_1UC1'
+    TYPE = 'mask'
+    UPLOADABLE_DESCRIPTION = 'Mask annotation data includes the actual mask (as a base64 '\
+        'encoded string), a width and a height.'
```
Comment (ahaith) [inline zegami_sdk/annotation.py:104]: base64 encoded PNG image.

Comment (Vaspra) [reply zegami_sdk/annotation.py]: Done

**Review by ahaith — APPROVED** [review]

**Pull request merged**